__tests__/app.test.js

```diff
@@ -130,4 +130,23 @@ describe("DELETE comment",()=>{
         .delete("/api/comments/1")
         .expect(204)
     })
-})
\ No newline at end of file
+})
+
+describe("GET users", ()=>{
+    test("Return all the available users",()=>{
+        return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then((responce) => {
+            expect(Array.isArray(responce.body.users)).toBe(true);
```

Comment:
There is an assertion that needs to be here 👀 If the response body.users is an empty array, the assertions in the .forEach won't run and the test will automatically pass 🫨 So we need an assertion to check that the array has a `length` property.